modules/application/src/main/java/edu/northeastern/truthtree/controller/basicInfo/BasicInfo.java

```diff
@@ -78,11 +80,10 @@ public JSONArray getBasicStatesInfo(@RequestParam
    */
   @Override
   @RequestMapping(value = "/api/counties", method = RequestMethod.GET)
-  public JSONArray getBasicCountiesInfo(@RequestParam
-                                                  (value = POPULATION_RANGE, required = false)
-                                                  int[] range) {
-    JSONArray response = this.service.getBasicCountiesInfo(range);
-    return response;
+  public ResponseEntity<PageDTO<LocationDTO>> getBasicCountiesInfo(
+        @RequestParam(value = POPULATION_RANGE, required = false) int[] range,
```

Comment:
Can you rename page to pageNumber? I guess other pagination supported apis have used pageNumber.

Reply:
addressed